package.json

```diff
@@ -14,7 +14,10 @@
     "build": "npm run scss && npm run assets",
     "prestart": "npm run build",
     "start": "node --experimental-modules src/server.js",
-    "dev": "nodemon --exec \"npm run lint && npm run start\" -e js,njk,scss --ignore dist/"
+    "dev": "TR_PORT=3999 nodemon --exec \"npm run lint && npm run start\" -e js,njk,scss --ignore dist/ --ignore cypress/",
+    "cy:run": "cypress run",
+    "test": "TR_PORT=3000 TR_PATH_PREFIX=trap-registration start-server-and-test start http://localhost:3000/trap-registration/health cy:run",
```

Comment:
Since the change to supply default values here, https://github.com/Scottish-Natural-Heritage/trap-registration/blob/develop/src/config.js#L5, and here, https://github.com/Scottish-Natural-Heritage/trap-registration/blob/develop/src/config.js#L9, I don't think we need to specify these env vars any more.

```diff
---    "test": "TR_PORT=3000 TR_PATH_PREFIX=trap-registration start-server-and-test start http://localhost:3000/trap-registration/health cy:run",
++    "test": "start-server-and-test start http://localhost:3000/trap-registration/health cy:run",
```

Reply:
Resolved in 323ddfb6e68f19f63bce8aad21b423e7a7f0fe73.